npm-shrinkwrap.json

```diff
@@ -23,9 +23,9 @@
       "resolved": "https://registry.npmjs.org/eyes/-/eyes-0.1.8.tgz"
     },
     "fh-db": {
-      "version": "1.1.0",
-      "from": "fh-db@1.1.0",
-      "resolved": "https://registry.npmjs.org/fh-db/-/fh-db-1.1.0.tgz",
+      "version": "1.1.1-114",
+      "from": "fh-db@1.1.1-114",
+      "resolved": "http://npm.skunkhenry.com/fh-db/-/fh-db-1.1.1-114.tgz",
```

Comment:
if the change in fh-db is ok, I say we publish it and re-generate the shrinkwrap file. This is ok for testing, but we don't want to release fh-mbaas-api with this.